conference-stepengine/src/main/java/org/tweetwallfx/conference/stepengine/steps/ShowSchedule.java

```diff
@@ -250,6 +292,34 @@ private Pane createSessionNode(final MachineContext context, final SessionData s
         return bpSessionPane;
     }
 
+    private Node createSpeakerImage(SpeakerImageProvider speakerImageProvider, Speaker speaker) {
+        var image = speakerImageProvider.getSpeakerImage(speaker);
+        var speakerImage = new ImageView(image);
+        speakerImage.getStyleClass().add("speakerImage");
+        speakerImage.setPreserveRatio(true);
+        if (image.getWidth() > image.getHeight()) {
+            speakerImage.setFitHeight(config.avatarSize);
+//            speakerImage.setX(image.getWidth()/2f - config.avatarSize / 2f);
+        } else {
+            speakerImage.setFitWidth(config.avatarSize);
+//            speakerImage.setY(image.getHeight()/2f - config.avatarSize / 2f);
+        }
+//        speakerImage.setFitWidth(config.avatarSize);
+//        speakerImage.setFitHeight(config.avatarSize);
```

Comment:
commented out lines can be removed
```suggestion
        } else {
            speakerImage.setFitWidth(config.avatarSize);
        }
```